framework/compose/kodein-di-framework-compose/src/jsMain/kotlin/org/kodein/di/compose/jsContext.kt

```diff
@@ -0,0 +1,6 @@
+package org.kodein.di.compose
```

Comment:
I'm rename jvmContext to jsContext in jsMain scope. This is git bug when calculate diff